elasticsearch/elasticsearch.yml

```diff
@@ -11,7 +11,7 @@ cluster.name: media-service
 # Node names are generated dynamically on startup, so you're relieved
 # from configuring them manually. You can tie this node to a specific name:
 #
-# node.name: "Franz Kafka"
```

Comment:
Indeed. I will miss googling the names of marvel characters while waiting
for the cluster to stabilise. Won't miss trying to figure out which node
needs killing from the AWS console.

On Mon, 7 Sep 2015 at 13:35 Sébastien Cevey <notifications@github.com>
wrote:

> In elasticsearch/elasticsearch.yml
> <https://github.com/guardian/grid/pull/1197#discussion_r38858447>:
>
> > @@ -11,7 +11,7 @@ cluster.name: media-service
> >  # Node names are generated dynamically on startup, so you're relieved
> >  # from configuring them manually. You can tie this node to a specific name:
> >  #
> > -# node.name: "Franz Kafka"
>
> Hence spoiling the only bit of fun during major incidents. What a tragedy!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/grid/pull/1197/files#r38858447>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396